spec/e2e/support/target-blank.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @typedef SimplifiedLink
+ *
+ * @prop {string=} innerText
+ * @prop {string=} href
+ * @prop {string=} className
+ * @prop {string=} target
+ */
+
+/** @type {import('puppeteer').Page} */
+/** @return {Promise<SimplifiedLink[]>} */
+async function getLinks(page) {
+  return await page.evaluate(() =>
+    Array.prototype.map.call(document.querySelectorAll('a'), (a) => {
```

Comment:
[`page.$$eval`](https://github.com/puppeteer/puppeteer/blob/v5.5.0/docs/api.md#pageevalselector-pagefunction-args) could be a good fit here as well.

Reply:
Used in 0008fef2fb6d4a067f3685472ec53bd441cec3b7